Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -92,7 +92,7 @@ class HomeModelController: HomeViewController {
         carouselView.layoutMargins = UIEdgeInsets(top: 0, left: 16, bottom: 0, right: 16)
         carouselView.titleLabel.text = title
 
-        for i in 0...2 {
+        for i in 0..<min(carouselEateries.count,3) {
```

Comment:
Is there a reason why this isn't just `carouselEateries.count`?

Reply:
Yea because we only want to show max 3 eateries in the carousel